### PR TITLE
PYTHON-3055 Release pymongoarrow 0.2

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#  Changelog
+
+## 0.2.0
```
Comment (behackett) [inline CHANGELOG.md:3]: Do we need to mention the bump to the required version of pyarrow?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#  Changelog
```
Comment (ShaneHarvey) [inline CHANGELOG.md:1]: There is already a changelog here: https://github.com/mongodb-labs/mongo-arrow/blob/main/bindings/python/docs/source/changelog.rst

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (blink1073) [reply CHANGELOG.md]: Added a note

File: CHANGELOG.md (hunk 2 above)
Comment (blink1073) [reply CHANGELOG.md]: Thanks, merged

**Review by behackett — APPROVED** [review]

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**